core/embed/projects/prodtest/cmd/prodtest_tropic.c

```diff
@@ -92,7 +367,81 @@ PRODTEST_CLI_CMD(
   .args = ""
 );
 
+PRODTEST_CLI_CMD(
+  .name = "tropic-lock-check",
+  .func = prodtest_tropic_lock_check,
+  .info = "Check whether Tropic is locked",
+  .args = ""
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-pair",
+  .func = prodtest_tropic_pair,
+  .info = "Pair with Tropic",
+  .args = ""
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-get-access-credential",
+  .func = prodtest_tropic_get_access_credential,
+  .info = "Get tropic access credential",
+  .args = ""
+);
 
+PRODTEST_CLI_CMD(
+  .name = "tropic-get-fido-masking-key",
+  .func = prodtest_tropic_get_fido_masking_key,
+  .info = "Get tropic FIDO masking key",
+  .args = ""
+);
 
+PRODTEST_CLI_CMD(
+  .name = "tropic-handshake",
+  .func = prodtest_tropic_handshake,
+  .info = "Perform handshake with Tropic",
+  .args = "<hex-data>"
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-send-command",
+  .func = prodtest_tropic_send_command,
+  .info = "Send command to Tropic",
+  .args = "<hex-data>"
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-certdev-read",
+  .func = prodtest_tropic_certdev_read,
+  .info = "Read the device's X.509 certificate",
+  .args = ""
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-certdev-write",
+  .func = prodtest_tropic_certdev_write,
+  .info = "Write the device's X.509 certificate",
+  .args = "<hex-data>"
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-certfido-read",
+  .func = prodtest_tropic_certfido_read,
+  .info = "Read the X.509 certificate for the FIDO key",
+  .args = ""
+);
+
+PRODTEST_CLI_CMD(
+  .name = "tropic-certfido-write",
+  .func = prodtest_tropic_certfido_write,
+  .info = "Write the X.509 certificate for the FIDO key",
```

Comment:
Nit: maybe mention "Tropic" here and above

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/5283/commits/243d090c3ea65d66c3944fb2b1dada231eff85d8.